### PR TITLE
Remove least common whitespaces in triple slash comments

### PR DESCRIPTION
**Pull request by qinezh**
@ansyral @vwxyzh @hellosnow @chenkennt

### CODE REVIEW

**Comment by qinezh** [comment]
#756

File: src/Microsoft.DocAsCode.Metadata.ManagedReference/Parsers/TripleSlashCommentModel.cs [hunk 1]
```diff
@@ -408,7 +407,41 @@ private string GetXmlValue(XPathNavigator node)
...
+            var normalized = new List<string>();
+
+            var preIndex = 0;
+            var leadingSpaces = from line in lines
+                where !string.IsNullOrWhiteSpace(line)
+                select line.TakeWhile(char.IsWhiteSpace).Count();
+
+            if (leadingSpaces.Any())
+            {
+                preIndex = leadingSpaces.Min();
```
Comment (hellosnow) [inline src/Microsoft.DocAsCode.Metadata.ManagedReference/Parsers/TripleSlashCommentModel.cs]: >preIndex [](start = 16, length = 8)

if preIndex is still 0 after min, return directly?

**Comment by hellosnow** [comment]
:shipit:

File: src/Microsoft.DocAsCode.Metadata.ManagedReference/Parsers/TripleSlashCommentModel.cs (hunk 1 above)
Comment (qinezh) [inline src/Microsoft.DocAsCode.Metadata.ManagedReference/Parsers/TripleSlashCommentModel.cs]: good!

**Comment by superyyrrzz** [comment]
👍

**Pull request merged**